src/main/java/frc/robot/commands/ArmMoveToShooterCommand.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotMap;
+import frc.robot.subsystems.ArmSystem;
+
+public class ArmMoveToShooterCommand extends Command {
+    private final ArmSystem armSystem;
+    private final PIDController pidController;
+
+    public ArmMoveToShooterCommand(ArmSystem armSystem) {
+        this.armSystem = armSystem;
+        pidController = new PIDController(0.07, 0, 0);
```

Comment:
since you create the PID controller in two different commands with the same gains, save those gains in constants in one place and use these constants when creating